src/client/Microsoft.Identity.Client/Internal/Logger/ILoggerAdapter.cs

```diff
@@ -12,6 +13,7 @@ internal interface ILoggerAdapter
         bool IsDefaultPlatformLoggingEnabled { get; }
         string ClientName { get; }
         string ClientVersion { get; }
+        IIdentityLogger MsalIdentityLogger { get; }
```

Comment:
Naming: IdenitityLogger i.e. could be either a legacy (callback) one or a new (interface) one. This class should not assume one or the other.